command/root.go

```diff
@@ -206,3 +204,70 @@ func determineBaseRepo(cmd *cobra.Command, ctx context.Context) (ghrepo.Interfac
 
 	return baseRepo, nil
 }
+
+func rootHelpFunc(command *cobra.Command, s []string) {
+	type helpEntry struct {
+		Title string
+		Body  string
+	}
+
+	coreCommandNames := []string{"issue", "pr", "repo"}
+	var coreCommands []string
+	var additionalCommands []string
+	for _, c := range command.Commands() {
+		if c.Short == "" {
+			continue
+		}
+		s := "  " + rpad(c.Name()+":", c.NamePadding()) + c.Short
+		if includes(coreCommandNames, c.Name()) {
+			coreCommands = append(coreCommands, s)
+		} else {
+			additionalCommands = append(additionalCommands, s)
+		}
+	}
+
+	helpEntries := []helpEntry{
+		{
+			"",
+			command.Long},
+		{"USAGE", `
+  # most gh commands need to be run in a directory containing a GitHub repository.
+  gh <command> [subcommand] [flags]`},
+		{"CORE COMMANDS", strings.Join(coreCommands, "\n")},
+		{"ADDITIONAL COMMANDS", strings.Join(additionalCommands, "\n")},
+		{"FEEDBACK", `
+  Fill out our feedback form <https://forms.gle/umxd3h31c7aMQFKG7>
+  Open an issue using “gh issue create -R cli/cli`},
+		{"FLAGS", strings.TrimRight(command.LocalFlags().FlagUsages(), "\n")},
+		{"EXAMPLES", `
+  $ cd your-repository
+  $ gh pr list
+  $ gh pr checkout 321`},
```

Comment:
```suggestion
  $ gh issue create
  $ gh repo clone
  $ gh pr checkout 321`},
```
I like the idea of using this space to hint that you should be in your repo but it feels a little out of place in a spot where we're introducing sample `gh` commands. I'd also love to use this space to show off some more variety in commands. I suggested some but we can talk about what'd be best! I love using this spot to show a command that uses the `321` syntax!